src/main/java/backend/resource/TurboIssue.java

```diff
@@ -126,7 +126,7 @@ public TurboIssue(String repoId, Issue issue) {
         this.description = issue.getBody() == null
             ? ""
             : issue.getBody();
-        this.updatedAt = Utility.dateToLocalDateTime(issue.getUpdatedAt());
+        this.updatedAt = replaceNull(Utility.dateToLocalDateTime(issue.getUpdatedAt()), this.createdAt);
```

Comment:
`dateToLocalDateTime` will fail if it gets a null value. Might need a manual check here.